src/components/Chat/InputMessageV2.tsx

```diff
@@ -388,7 +388,7 @@ const InputMessageV2 = ({
         '',
         type === ANONYMOUS
           ? `Switch back to regular chat?\nMessage ${username} using your username.`
-          : 'Switch to anonymous chat?\nMessage this user anonymously instead.',
+          : 'Switch to anonymous chat?\nMessage this user in Incognito Mode instead.',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the text prompt from 'Message this user anonymously instead.' to 'Message this user in Incognito Mode instead.' is more intuitive and user-friendly. It makes it clear that the user's identity will be hidden during the chat, which aligns with common terminology used in other applications.
```diff
- 'Switch to anonymous chat?\nMessage this user anonymously instead.',
+ 'Switch to anonymous chat?\nMessage this user in Incognito Mode instead.',
```
However, please ensure that the term "Incognito Mode" is consistently used across the application for anonymous chats to avoid any confusion.


<!-- This is an auto-generated comment by OSS CodeRabbit -->